src/python/pants/util/contextutil.py

```diff
@@ -76,6 +86,10 @@ def setenv(key, val):
       setenv(key, val)
 
 
+def _copy_env(env):
```

Comment:
I don't think `decoding` is idempotent, so this method will only work the first time you call it? In which case, maybe call it `_decode_and_copy_env` or something, so that somebody doesn't try to use it to just copy.

Reply:
fixed